src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md

```diff
@@ -0,0 +1,287 @@
+integration, your providers and customers will show up in different ways
+on your Stripe platform account. The main ways this happens is through
+Stripe Connect accounts and Stripe Customers.
+
+The Sharetribe Stripe integration uses Stripe Custom Connect accounts
+for providers. When a provider first creates a listing, they need to
+onboard to Stripe Connect before they can receive payouts. For
+customers, on the other hand, a Stripe Customer does not need to be
+created by default.
+
+## Stripe for providers: Stripe Connect Onboarding
+
+When your marketplace handles payments, it is important to have a
+provider verification process where providers can enter all necessary
+and required information for them to receive payments. Provider
+verification is
+[a requirement from Stripe's side](https://support.stripe.com/questions/know-your-customer-obligations)
+for identity verification, risk assessment, and avoiding money
+laundering and other types of financial crime.
+
+Regulatory aspects of provider onboarding can be challenging and
+changing rapidly. Stripe Connect Onboarding provides ready tools for
+meeting the requirements and reducing the operational complexity of
+self-managing the onboarding flow and identity verification.
+
+For both Sharetribe Web Template and custom implementations, your
+marketplace will first need to set up Stripe for payments and enable
+Connect Onboarding. You can review the
+[instructions in our Help Center](https://www.sharetribe.com/help/en/articles/8413086-how-to-set-up-stripe-for-payments-on-your-marketplace)
+for more details.
+
+### Stripe Connect Custom Accounts
+
+Sharetribe uses Stripe Connect Custom Accounts as the default payment
+integration. These enable your sellers to process payments through your
+marketplace. Stripe Connect also has other types of accounts, such as
+Standard and Express, but those are not compatible with the Sharetribe
+Stripe integration.
+
+When you use the Sharetribe Stripe integration, you need to create
+Custom Connect accounts for your users through the Sharetribe
+Marketplace API
+[Stripe Account creation endpoint](https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-account).
+It is not possible to associate an existing Custom Connect account with
+a Sharetribe user profile.
+
+When you need to update the Stripe Account, you can either use the
+Marketplace API
+[Stripe Account updating endpoint](https://www.sharetribe.com/api-reference/marketplace.html#update-stripe-account)
+or directly through
+[Stripe API](https://docs.stripe.com/api/accounts/update).
+
+### Fetching an existing Stripe Connect Account
+
+If the user already has a Stripe account, the
+[currentUser resource](https://www.sharetribe.com/api-reference/marketplace.html#currentuser-resource-format)
+has a _stripeConnected_ boolean flag set to _true._ To fetch the
+provider's existing Stripe Connect account details, we can use the
+Marketplace API
```

Comment:
Suggestion: consider using either `user` or `provider` in this paragraph for consistence